bot/kodiak/queries/__init__.py

```diff
@@ -993,18 +993,39 @@ async def get_open_pull_requests(
             params["base"] = base
         if head is not None:
             params["head"] = head
+
+        params["per_page"] = "100"
+        open_prs = []
+
         async with self.throttler:
```

Comment:
We need to wrap each API call (`self.session.get`) with this context manager.

So it would be:

```python
while page:
    with self.throttler:
        res = await self.session.get(...)
```

Reply:
on it.